test/jdk/sun/security/ssl/SSLKEYLOGFILE/TLSKeyLoggerTest.java

```diff
@@ -0,0 +1,277 @@
+/*
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+// SunJSSE does not support dynamic system properties, no way to re-use
+// system properties in samevm/agentvm mode.
+
+/*
+ * @test
+ * @bug 8262880
+ * @summary Add support for the NSS Key Log Format for SSL/TLS keys
+ * @library /javax/net/ssl/templates
+ * @library TLSKeyLogger
+ * @build java.base/sun.security.ssl.TLSKeyLogger
+ *
+ * @run main/othervm TLSKeyLoggerTest TLSv1.3
+ * run main/othervm TLSKeyLoggerTest TLSv1.2
```

Comment:
Missing `@` sign before `run`.

Reply:
This is still prototype and under development, I haven't implemented the TLSv1.2 side yet.

The logging style/approach is still TBD, but the location of the hooks likely won't change.